docs/configuration/index-config.md

```diff
@@ -186,7 +186,8 @@ The `datetime` type handles dates and datetimes. Quickwit supports multiple inpu
 
 - `unix_timestamp`: parse Unix timestamp values. Timestamp values come in different precision, namely: `seconds`, `milliseconds`, `microseconds`, or `nanoseconds`. Quickwit is capable of inferring the precision from the value. Because of this feature, Quickwit only supports timestamp values ranging from `13 Apr 1972 23:59:55` to `16 Mar 2242 12:56:31`.
 
-When a `datetime` field is stored as a fast field, the `precision` parameter indicates the precision used to truncate the values before encoding and compressing them. The `precision` parameter can take the following values: `seconds`, `milliseconds`, `microseconds`.
+When a `datetime` field is stored as a fast field, the `precision` parameter indicates the precision used to truncate the values before encoding which improves compression. The `precision` parameter can take the following values: `seconds`, `milliseconds`, `microseconds`.
+Internally `datetime` is stored as microseconds, truncation does only mean zeroing the precision.
```

Comment:
I think this needs to be a warning or info box, we accept any value but internally we store at microseconds. precision is only in play for fastfield.